spec/system/add_update_phone_number_spec.rb

```diff
@@ -7,12 +7,17 @@
   let(:user) { create(:user, :confirmed, organization: organization) }
   let!(:auth_settings) { create(:auth_setting, organization: organization) }
   let(:decidim_half_signup_admin) { Decidim::HalfSignup::AdminEngine.routes.url_helpers }
+  let(:sms_gateway_service) { "Decidim::Verifications::Sms::ExampleGateway" }
   let(:phone) { "4578878784" }
 
   before do
     sign_in user
     switch_to_host(organization.host)
     visit decidim.account_path
+
+    Decidim.configure do |config|
```

Comment:
Prefer to define mock for `Decidim.configure.sms_gateway_service` 

Example : `allow(Decidim.config.sms_gateway_service).to receive(:present?).and_return(true)`